src/client/gitlab.ts

```diff
@@ -79,7 +79,10 @@ export const callGitlab = async (
   }
 
   if (resp.status >= 300) {
-    console.warn(`Gitlab client received a status code of ${resp.status} while fetching ${path}`);
+    const msg = await resp.json();
```

Comment:
@ezhong2 have you tested this? I know for some other APIs (Bitbucket for example) error responses come back as text, not json

Reply:
Not sure if the `resp` is actually ever going to be in text format, but I saw that there was a check for that after this line so I added it! good catch 👍